### ISSUE TITLE
Web Accessibility standards

### ISSUE THREAD

**Issue by Baranitharan84** [jackocnr/intl-tel-input#800]
I'm trying to use your library for our project so I referred your demo page to our internal accessibility team to check whether the **intl-tel-input** is compliant with accessibility standards. 

They said that it is keyboard accessible, however, when the dropdown list is tabbed through it does not announce the country (flag) or the phone numbers in the list.
A screen reader user does not know what is in the dropdown listbox in order to select one. Also, some ARIA roles, states or properties are invalid.

Do you have any backlog item to make it compliance with accessibility in near future ?

**Comment by jacob-kruger-work** [comment]
I myself am a full-time, senior web application developer, and, have been for over 20 years, but, I am also 100% blind myself, and, sorry, but, trying to work with intl-tel input implementation in a couple of different pages in our primary project, working with NVDA, jaws, and, even MS narrator, across multiple browsers - chrome, firefox, waterfox, brave, etc., the intl-tel input tweak seems to make the telephone number input boxes too inconsistent - every now and then I will be able to use forms of virtual buffer framework navigation to locate the list of countries, but, not always, even when working with the same combination of assistive technology/browser, and, almost the worst is that even if I manage to select a country from the list, when I then try to edit the actual phone number, it seems to just be discarding, or blocking my keystrokes, almost all the time, or undoing any changes, so, sorry, but, putting aria attributes in place literally means nothing if you are not allowing assistive technology to still interact with a control or element in the way it interprets it as being required. And, I tried the above with both the straight javascript implementation, as well as the jQuery version - same result

**Comment by jackocnr** [comment]
Hi @jacob-kruger-work, thanks so much for your message. I would love to make this plugin more accessible, but as I mentioned before, unfortunately I personally do not have the time to work on this right now. If you have time, would you be able to check out the work in [this PR](https://github.com/jackocnr/intl-tel-input/pull/1002) and see if that helps at all? Or perhaps to offer advice on how we could improve the code, or even submit a PR yourself? Thanks so much.

**Comment by agarzola** [comment]
Hi, @jackocnr. I would suggest keeping this issue open. Doing so will increase visibility for anyone running into the same issue, as well as folks who might be interested in making a contribution to the library.

**Comment by jackocnr** [comment]
Good idea.

**Comment by jacob-kruger-work** [comment]
Can you pass on a specific branch cloning URL?

**Comment by jackocnr** [comment]
I just merged this PR: https://github.com/jackocnr/intl-tel-input/pull/1002 into master and released a new version: v16.0.15 - please could you check if that has solved the issue you are experiencing?

**Comment by jacob-kruger-work** [comment]
Actually yes, it now seems very usable along with NVDA screenreader, in 
chrome, firefox, brave and waterfox.


Jacob Kruger
+2782 413 4791
"Resistance is futile...but, acceptance is versatile..."

**Comment by jacob-kruger-work** [comment]
Ok, and, just tested it along with both MS narrator - windows 10's own 
built-in screenreader, and, jaws 2019, under waterfox, and, it also 
worked well, so, thank you.


Jacob Kruger
Blind Biker
"Resistance is futile...but, acceptance is versatile..."

**Comment by jackocnr** [comment]
@jacob-kruger-work thank you so much for testing this. So glad it's working now :)
